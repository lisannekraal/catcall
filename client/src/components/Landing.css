.landing {
  display: flex;
  flex-direction: column;
}

.landing-cover {
  min-height: 450px;
  background: url('https://res.cloudinary.com/serendatapy/image/upload/v1611397206/catcalls/sam-haddad-StgZKNLVeJg-unsplash-crop_figayu.jpg') center center fixed no-repeat;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

@media screen and (min-height: 650px) {
  .landing-cover {
    height: 95vh;
  }
}

.header {
  max-width: 800px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-start;
  color: white;
  height: 100%;
  padding: 65px 20px 20px 20px;
}

.header-title {
  font-size: 7rem;
  text-shadow: 3px 3px black;
}

@media screen and (max-width: 400px) {
  .header-title {
    font-size: 6rem;
  }
}

.header-buttons {
  display: flex;
  align-items: flex-start;
}

.header-buttons a {
  text-decoration: none;
  margin-right: auto;
}

.header-buttons a p{
  border-radius: 8px;
  border-width: 2px;
  border-color: black;
  border-top-color: rgb(118,118,118);
  border-left-color: rgb(118,118,118);
  border-style: outset;
  background-color: rgb(245, 37, 89);
  color: white;
  padding: 10px 15px;
  font-size: 1.2rem;
}

@media screen and (max-width: 350px) {
.header-buttons a {
  width: 100%;
    text-align: center;
  }
}

.header-disclaimer {
  margin-top: 5px;
  font-size: 0.9rem;
  font-weight: 300;
}



.landing-why {
  padding: 20px;
  margin: 0 auto 50px auto;
  display: flex;
  flex-wrap: wrap;
  justify-content: flex-start;
}

.why-title {
  font-size: 7rem;
  color: rgb(29, 113, 183);
}

.why-content {
  padding: 50px 0 0 50px;
  max-width: 500px;
}

@media screen and (max-width: 800px) {
  .landing-why {
    max-width: 500px;
  }
  .why-content {
    padding: 0;
  }
}

.landing-community {
  background-color: rgb(245, 37, 89);
}

.community-content {
  padding: 60px;
  text-align: center;
  display: flex;
  align-items: center;
  flex-direction: column;
}

.community-title {
  font-size: 4rem;
  color: white;
}

.community-content button {
  border-radius: 8px;
  background-color: black;
  color: white;
  padding: 10px 15px;
  margin-top: 20px;
}

.about-content button {
  border-radius: 8px;
  background-color: black;
  color: white;
  padding: 10px 15px;
  margin-top: 20px;
}

button p {
  margin: 0 0;
  font-size: 1.2rem;
}

.landing-about {
  padding: 60px;
  margin: 0 auto 50px auto;
  display: flex;
  flex-wrap: wrap-reverse;
  justify-content: flex-start;
}

.about-title {
  font-size: 5rem;
  color: rgb(29, 113, 183);
  max-width:  500px;
}

.about-content {
  padding: 50px 50px 0 0;
  max-width: 500px;
}

.landing-widget-container {
  max-width:100%;
  width: 550px;
  height: 550px;
}

@media screen and (max-width: 1100px) {
  .landing-about {
    max-width: 500px;
  }
  .about-title {
    font-size: 4.5rem;
  }
  .about-content {
    padding: 0;
  }
}

@media screen and (max-width: 450px) {
  .about-title {
    font-size: 3rem;
  }
  .landing-widget-container {
    height: 325px;
  }
  .community-content,.landing-about {
    padding: 30px;
  }
}

@media screen and (max-width: 350px) {
  .community-content,.landing-about {
    padding: 30px;
  }

  .about-title {
    font-size: 2.5rem;
  }
  .landing-widget-container {
    height: 250px;
  }

}

.help-link {
  color: white;
}
