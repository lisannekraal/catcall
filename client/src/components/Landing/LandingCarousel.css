.landing-cities {
  display: flex;
  justify-content: center;
  margin: 50px auto;
  max-width: 400px;
}

.carousel-content {
  display: flex;
}
.carousel-overflow {
  overflow: visible !important;
}

.city {
  margin: 10px;
  text-align: center;
}

.city-logo {
  height: 80px;
  width: 80px;
  margin-left: 20px;
}

.city-logo img {
  height: 100%;
  width: 100%;
}

.city-name {
  font-weight: 700;
}

.city-description {
  font-style: italic;
  margin-left: 5px;
}