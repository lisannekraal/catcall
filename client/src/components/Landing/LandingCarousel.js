import React from 'react';
import { useTranslation } from 'react-i18next';
import Carousel from 'react-material-ui-carousel'
import LocationOnIcon from '@material-ui/icons/LocationOn';
import LogoRot from '../../assets/catcallsofrot.png';
import LogoUtr from '../../assets/catcallsofutr.png';
import LogoNyc from '../../assets/catcallsofnyc.png';
import LogoGrunn from '../../assets/catcallsofgrunn.png';
import { v4 as uuidv4 } from 'uuid';

function LandingCarousel({ citiesDisplayed }) {

  const { t } = useTranslation(['landing']);

  const cities = [
      {
        name: 'R\'DAM',
        logo: LogoRot,
        status: t('cities.status-soon', 'default')
      },
      {
        name: 'UTRECHT',
        logo: LogoUtr,
        status: t('cities.status-soon', 'default')
      },
      {
        name: 'NEW YORK',
        logo: LogoNyc,
        status: t('cities.status-soon', 'default')
      },
      {
        name: 'ANTWERP',
        logo: LogoNyc,
        status: t('cities.status-soon', 'default')
      },
      {
        name: 'BERLIN',
        logo: LogoNyc,
        status: t('cities.status-soon', 'default')
      },
      {
        name: 'GRONINGEN',
        logo: LogoGrunn,
        status: t('cities.status-soon', 'default')
      }
  ];

  function citiesToDisplay(citiesPerRow) {
    let count = 0;
    let row = 0;
    let citiesList = [[]];

    for(let i = 0; i < cities.length; i++) {
      let city = cities[i];

      if(count < citiesPerRow){
        citiesList[row].push(city)
        count++;
      }
      else{
        citiesList.push([]);
        row++;
        citiesList[row].push(city);
        count = 1;
      }
    }
    return citiesList;
  }


  function Group({ group }) {
    return (
      <div className="carousel-content">
        {
          group.map((city, i) =>
            <City key={uuidv4()} city={city} />
          )
        }
      </div>
    )
  }

  function City({ city }) {
    return (
      < div className="city" >
        <div>
          <div className="city-logo">
            <img alt="city-logo" src={city.logo}></img>
          </div>
        </div>
        <div>
          <div className="city-name catcall-font">
            <LocationOnIcon fontSize="small" />
            {city.name}
          </div>
          <p className="city-description">{city.status}</p>
        </div>
      </div >
    )
  }

  const citiesInRows = citiesToDisplay(citiesDisplayed);

  return (
    <div>
      <div className="landing-cities">
        <Carousel className="carousel-overflow">
          {
            citiesInRows.map((group, i) =>
              <Group key={uuidv4()} group={group} />
            )
          }
        </Carousel>
      </div>

    </div>
  )
}

export default LandingCarousel
