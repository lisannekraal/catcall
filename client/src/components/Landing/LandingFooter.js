// import React from "react";
import InstagramIcon from '@material-ui/icons/Instagram';
import MailOutlineIcon from '@material-ui/icons/MailOutline';

function LandingFooter() {
  return (
    <div className="footer">
      <div className="footer-content">
        <div className="footer-div">
          <div className="footer-title catcall-font">ABOUT THIS MAP APPLICATION</div>
          <p>
            This is a project for Catcalls of Amsterdam by Lisanne Kraal. Inspired by the young activists, she contacted the organization to help them mapping their enormous database in an interactive way. This is a first version of a project under construction. Any feedback or want to contribute? Contact us.
          </p>
        </div>
        <div className="footer-div">
          <div className="footer-title catcall-font">CONTACT</div>
          {/* <div>Catcalls of Amsterdam</div> */}
          <p>Catcalls of Amsterdam</p>

          <a style={{ textDecoration: 'none', color: 'white', display: 'flex', alignItems: 'center' }} href="https://www.instagram.com/catcallsofams/" target="_blank" rel="noreferrer nofollow">
            <InstagramIcon fontSize='small' />
            <div>PM on Instagram</div>
          </a>

          <a style={{ textDecoration: 'none', color: 'white', display: 'flex', alignItems: 'center' }} href="mailto:catcallsofams@gmail.com">
            <MailOutlineIcon fontSize='small' />
            <div>Send us an email</div>
          </a>

        </div>
      </div>
    </div>
  )
}

export default LandingFooter;