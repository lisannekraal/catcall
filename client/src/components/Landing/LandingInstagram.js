import React from 'react';

function LandingInstagram() {
  return (
      <iframe title="catcall photos" src="https://snapwidget.com/embed/900599" className="snapwidget-widget" allowtransparency="true" frameBorder="0" scrolling="no" style={{ border: 'none', overflow: 'scroll', height: '100%', width: '100%' }}></iframe>
  )
}

export default LandingInstagram;
