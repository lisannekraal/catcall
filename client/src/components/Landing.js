import React from "react";
import { Link, useHistory } from 'react-router-dom';
import './Landing.css';

import Footer from './Footer';

import Carousel from 'react-material-ui-carousel'

import LogoRot from '../assets/catcallsofrot.png';
import LogoUtr from '../assets/catcallsofutr.png';
import LogoNyc from '../assets/catcallsofnyc.png';
import LogoGrunn from '../assets/catcallsofgrunn.png';

function Landing () {

  const cities = [
    [
        { name: 'ROTTERDAM',
        logo: LogoRot,
        status: 'coming soon'},
        { name: 'UTRECHT',
        logo: LogoUtr,
        status: 'coming soon'},
        { name: 'NEW YORK',
        logo: LogoNyc,
        status: 'coming soon'}
    ],
    [
        { name: 'ANTWERP',
        logo: LogoNyc,
        status: 'coming soon'},
        { name: 'BERLIN',
        logo: LogoNyc,
        status: 'coming soon'},
        { name: 'GRONINGEN',
        logo: LogoGrunn,
        status: 'coming soon'}
    ]
  ];

  return (
    <div data-testid="landing" className="landing">
      <div className="landing-cover">

        <div className="header">
          <div className="header-title">Chalk back.</div>
          <div className="header-buttons">
            <Link to={{
              pathname: '/catcalls',
              state: { dialog: 'This is an overview of all catcalls reported. Disclaimer: this application contains violent language and words of racism, sexual harassment and hate speach. Are you younger than 18 years old? Ask your guardian.'}
            }}>
              <p>View Catcalls of Amsterdam</p>
            </Link>
            <Link to="/catcalls/new">
              <p>Report a new catcall</p>
            </Link>
          </div>
          <div className="header-disclaimer">
            This initiative is standing up against street harassment and does not replace reporting your issue to the police. Are you currently in danger or need help right away? <a class="help-link" href="/help">Click here</a>.
          </div>
        </div>

      </div>


      <div className="landing-cities">
        <Carousel>
          {
            cities.map( (group, i) => 
            <div className="carousel-content">
              {
                group.map( (item, i) => 
                <div className="city">
                  <div>
                    <div className="city-logo">
                      <img src={item.logo}></img>
                    </div>
                  </div>
                  <div>
                    <div className="city-name"><i className="fas fa-map-marker-alt"></i> {item.name}</div>
                    <div className="city-description">{item.status}</div>
                  </div>
                </div>
                )
              }
            </div>
            )
          }
        </Carousel>
      </div>



      <div className="landing-why" id="about">
        <div className="why-title">Why</div>
        <div className="why-content">
          <p>A catcall is basically everything sexist, insulting, queerphobic, racist or else shouted after and at you in the streets. This also includes wolf whistling, being followed, not accepting a “no” and encompasses a host of other behaviours.  It is a form of gender-based violence.</p>
          <p>Worldwide, youth-led movements - like Catcalls of Amsterdam - are committed to ending gender-based street harassment with public chalk art, digital media and education. We write stories of harassment word-for-word in the spots where they happened alongside the hashtag #stopstreetharassment using sidewalk chalk and then post on social media to spur dialogue and story sharing</p>
          <p>Why do we do this? On the one hand, we want to claim back the space of the harassment for the victim and give them a voice. On the other hand we want to show those passing our chalk art that catcalling happens everywhere and engage them in a discussion about it. Catcalling is still too normal in our society and we want to change that, together with you. </p>
        </div>
      </div>

      <div className="landing-community">
        <div className="community-content">
          <div className="community-title">Get involved</div>
          <div style={{margin: '30px auto', width: '550px', height: '550px'}}>
            <iframe src="https://snapwidget.com/embed/900599" class="snapwidget-widget" allowtransparency="true" frameborder="0" scrolling="no" style={{border: 'none', overflow: 'scroll', height: '100%', width: '100%'}}></iframe>
          </div>
          <a href="https://www.instagram.com/catcallsofams/" target="_blank" rel="noreferrer nofollow">
            <button><p>Join the community on Instagram</p></button>
          </a>
        </div>
      </div>

      <div className="landing-about">

        <div className="about-content">
          <p>Catcalls of Amsterdam is founded in December 2017, after Ambrien found the account Catcalls of New York on Instagram. After talking with Sophie Sandberg, its founder, she decided to join the movement and start chalking in Amsterdam.</p>
          <p>In the meantime our team grew and is organised by four people: Ambrien, Margriet, Paula and Zessiah. Together with 150 other initiatives worldwide, we 'chalk back'. Our goal is to provide a place to share experiences on street harassment and use people's stories to create awereness around catcalling and denormalize it.</p>
          <p>With this application outside of Instagram, we show our catcalling database on the map and make it easier for any victim to report their experience. Check it out and contact us if you have any questions.</p>
          <a href="/help" rel="noreferrer nofollow">
            <button><p>Go to help page and FAQ</p></button>
          </a>
        </div>
        <div className="about-title">About Catcalls of Amsterdam</div>
      </div>

      <Footer />
    </div>
  );
}


export default Landing;