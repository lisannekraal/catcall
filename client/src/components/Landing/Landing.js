import React, { lazy, Suspense } from "react";
import { useTranslation } from 'react-i18next';
import { Link } from 'react-router-dom';
import LandingFooter from './LandingFooter';
import LandingCarousel from "./LandingCarousel";
import Hidden from '@material-ui/core/Hidden';
const LandingInstagram = lazy(()=> import("./LandingInstagram"));

function Landing() {

  const { t } = useTranslation(['landing']);

  return (
    <div data-testid="landing" className="landing">
      <div className="landing-cover">

        <div className="header">
          <div className="header-title catcall-font">{t('title', 'default')}</div>
          <div className="header-buttons">
            <Link to={{
              pathname: '/catcalls',
              state: { dialog: t('disclaimer', 'default') }
            }}>
              <p>{t('button.view', 'default')}</p>
            </Link>
            <Link to="/catcalls/new">
              <p>{t('button.report', 'default')}</p>
            </Link>
          </div>
          <p className="header-disclaimer">
            {t('subtitle', 'default')} <a className="help-link" href="/help">{t('subtitle-cta', 'default')}</a>.
          </p>
        </div>

      </div>
      <Hidden smUp>
        <LandingCarousel citiesDisplayed={2} />
      </Hidden>
      <Hidden xsDown lgUp>
        <LandingCarousel citiesDisplayed={3} />
      </Hidden>
      <Hidden mdDown>
        <LandingCarousel citiesDisplayed={6} />
      </Hidden>


      <div className="landing-why" id="about">
        <div className="why-title catcall-font">{t('why.title', 'default')}</div>
        <div className="why-content">
          <p>{t('why.p1', 'default')}</p>
          <p>{t('why.p2', 'default')}</p>
          <p>{t('why.p3', 'default')}</p>
        </div>
      </div>

      <div className="landing-community">
        <div className="community-content">
          <div className="community-title catcall-font">{t('social.title', 'default')}</div>
          <div className="landing-widget-container" style={{ margin: '30px auto' }}>
          <Suspense fallback={<div>Loading...</div>}>
            <LandingInstagram/>
          </Suspense>
          </div>
          <a href="https://www.instagram.com/catcallsofams/" target="_blank" rel="noreferrer nofollow">
            <button className="normal-font"><p>{t('button.social', 'default')}</p></button>
          </a>
        </div>
      </div>

      <div className="landing-about">

        <div className="about-content">
          <p>{t('about.p1', 'default')}</p>
          <p>{t('about.p2', 'default')}</p>
          <p>{t('about.p3', 'default')}</p>
          <a href="/help" rel="noreferrer nofollow">
            <button className="normal-font"><p>{t('button.faq', 'default')}</p></button>
          </a>
        </div>
        <div className="about-title catcall-font">{t('about.title', 'default')}</div>
      </div>

      <LandingFooter />
    </div>
  );
}

export default Landing;