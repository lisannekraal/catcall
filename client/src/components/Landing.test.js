import Landing from './Landing';
import { render, screen } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

const snap = 

describe ('Landing tests:', ()=>{

  test('displays landing page correctly', async ()=>{
    const landing = TestRenderer
      .create(<BrowserRouter><Landing/></BrowserRouter>)
      .toJSON();
    expect(landing).toMatchSnapshot();
  });

});