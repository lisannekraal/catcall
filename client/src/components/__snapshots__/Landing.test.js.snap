// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing tests: displays landing page correctly 1`] = `
<div
  className="landing"
  data-testid="landing"
>
  <div
    className="landing-cover"
  >
    <div
      className="header"
    >
      <div
        className="header-title"
      >
        Chalk back.
      </div>
      <div
        className="header-buttons"
      >
        <a
          href="/catcalls"
          onClick={[Function]}
        >
          <p>
            View Catcalls of Amsterdam
          </p>
        </a>
        <a
          href="/catcalls/new"
          onClick={[Function]}
        >
          <p>
            Report a new catcall
          </p>
        </a>
      </div>
      <div
        className="header-disclaimer"
      >
        Disclaimer: this application contains violent language and words of racism, sexual harassment and hate speach. Are you younger than 18 years old? Ask your guardian.
      </div>
    </div>
  </div>
  <div
    className="landing-cities"
  >
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-rot"
            src="catcallsofrot.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           ROTTERDAM
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-utr"
            src="catcallsofutr.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           UTRECHT
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-nyc"
            src="catcallsofnyc.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           NEW YORK
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-ant"
            src="catcallsofnyc.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           ANTWERP
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-ber"
            src="catcallsofnyc.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           BERLIN
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-grunn"
            src="catcallsofgrunn.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           GRONINGEN
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
  </div>
  <div
    className="landing-why"
    id="about"
  >
    <div
      className="why-title"
    >
      Why.
    </div>
    <div
      className="why-content"
    >
      <p>
        A catcall is basically everything sexist, insulting, queerphobic, racist or else shouted after and at you in the streets. This also includes wolf whistling, being followed, not accepting a “no” and encompasses a host of other behaviours.  It is a form of gender-based violence.
      </p>
      <p>
        Worldwide, youth-led movements - like Catcalls of Amsterdam - are committed to ending gender-based street harassment with public chalk art, digital media and education. We write stories of harassment word-for-word in the spots where they happened alongside the hashtag #stopstreetharassment using sidewalk chalk and then post on social media to spur dialogue and story sharing
      </p>
      <p>
        Why do we do this? On the one hand, we want to claim back the space of the harassment for the victim and give them a voice. On the other hand we want to show those passing our chalk art that catcalling happens everywhere and engage them in a discussion about it. Catcalling is still too normal in our society and we want to change that, together with you. 
      </p>
    </div>
  </div>
  <div
    className="landing-community"
  >
    <div
      className="community-content"
    >
      <div
        className="community-title"
      >
        Get involved
      </div>
      <div
        className="community-text"
      >
        See the social media channel for actions of Catcalls of Amsterdam: the stories of catcalls being reported to the organization and pictures of thse chalk back actions. Join the discussion and share the chalked catcalls so we can create awareness.
      </div>
      <a
        href="https://www.instagram.com/catcallsofams/"
        rel="noreferrer nofollow"
        target="_blank"
      >
        <button>
          <p>
            Join the community on Instagram
          </p>
        </button>
      </a>
    </div>
  </div>
  <div
    className="landing-about"
  >
    <div
      className="about-content"
    >
      <p>
        Catcalls of Amsterdam is founded in December 2017, after Ambrien found the account Catcalls of New York on Instagram. After talking with Sophie Sandberg, its founder, she decided to join the movement and start chalking in Amsterdam.
      </p>
      <p>
        In the meantime our team grew and is organised by four people: Ambrien, Margriet, Paula and Zessiah. Together with 150 other initiatives worldwide, we 'chalk back'. Our goal is to provide a place to share experiences on street harassment and use people's stories to create awereness around catcalling and denormalize it.
      </p>
      <p>
        With this application outside of Instagram, we show our catcalling database on the map and make it easier for any victim to report their experience. Check it out and contact us if you have any questions.
      </p>
    </div>
    <div
      className="about-title"
    >
      About Catcalls of Amsterdam.
    </div>
  </div>
  <div
    className="footer"
  >
    <div
      className="footer-content"
    >
      <div
        className="footer-div"
      >
        <div
          className="footer-title"
        >
          ABOUT THIS MAP APPLICATION
        </div>
        <div>
          This is a project for Catcalls of Amsterdam by Lisanne Kraal. Inspired by the young activists, she contacted the organization to help them mapping their enormous database in an interactive way. This is a first version of a project under construction. Any feedback or want to contribute? Contact us.
        </div>
      </div>
      <div
        className="footer-div"
      >
        <div
          className="footer-title"
        >
          CONTACT
        </div>
        <div>
          Catcalls of Amsterdam
        </div>
        <div>
          <a
            href="https://www.instagram.com/catcallsofams/"
            rel="noreferrer nofollow"
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            <i
              className="fab fa-instagram"
            />
             PM on Instagram
          </a>
        </div>
        <div>
          <a
            href="mailto:catcallsofams@gmail.com"
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
          >
            <i
              className="far fa-envelope"
            />
             Send an email
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Landing tests: displays page correctly 1`] = `
<div
  className="landing"
  data-testid="landing"
>
  <div
    className="landing-cover"
  >
    <div
      className="header"
    >
      <div
        className="header-title"
      >
        Chalk back.
      </div>
      <div
        className="header-buttons"
      >
        <a
          href="/catcalls"
          onClick={[Function]}
        >
          <p>
            View Catcalls of Amsterdam
          </p>
        </a>
        <a
          href="/catcalls/new"
          onClick={[Function]}
        >
          <p>
            Report a new catcall
          </p>
        </a>
      </div>
      <div
        className="header-disclaimer"
      >
        Disclaimer: this application contains violent language and words of racism, sexual harassment and hate speach. Are you younger than 18 years old? Ask your guardian.
      </div>
    </div>
  </div>
  <div
    className="landing-cities"
  >
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-rot"
            src="catcallsofrot.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           ROTTERDAM
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-utr"
            src="catcallsofutr.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           UTRECHT
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-nyc"
            src="catcallsofnyc.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           NEW YORK
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-ant"
            src="catcallsofnyc.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           ANTWERP
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-ber"
            src="catcallsofnyc.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           BERLIN
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
    <div
      className="city"
    >
      <div>
        <div
          className="city-logo"
        >
          <img
            alt="logo-grunn"
            src="catcallsofgrunn.png"
          />
        </div>
      </div>
      <div>
        <div
          className="city-name"
        >
          <i
            className="fas fa-map-marker-alt"
          />
           GRONINGEN
        </div>
        <div
          className="city-description"
        >
          coming soon
        </div>
      </div>
    </div>
  </div>
  <div
    className="landing-why"
    id="about"
  >
    <div
      className="why-title"
    >
      Why.
    </div>
    <div
      className="why-content"
    >
      <p>
        A catcall is basically everything sexist, insulting, queerphobic, racist or else shouted after and at you in the streets. This also includes wolf whistling, being followed, not accepting a “no” and encompasses a host of other behaviours.  It is a form of gender-based violence.
      </p>
      <p>
        Worldwide, youth-led movements - like Catcalls of Amsterdam - are committed to ending gender-based street harassment with public chalk art, digital media and education. We write stories of harassment word-for-word in the spots where they happened alongside the hashtag #stopstreetharassment using sidewalk chalk and then post on social media to spur dialogue and story sharing
      </p>
      <p>
        Why do we do this? On the one hand, we want to claim back the space of the harassment for the victim and give them a voice. On the other hand we want to show those passing our chalk art that catcalling happens everywhere and engage them in a discussion about it. Catcalling is still too normal in our society and we want to change that, together with you. 
      </p>
    </div>
  </div>
  <div
    className="landing-community"
  >
    <div
      className="community-content"
    >
      <div
        className="community-title"
      >
        Get involved
      </div>
      <div
        className="community-text"
      >
        See the social media channel for actions of Catcalls of Amsterdam: the stories of catcalls being reported to the organization and pictures of thse chalk back actions. Join the discussion and share the chalked catcalls so we can create awareness.
      </div>
      <a
        href="https://www.instagram.com/catcallsofams/"
        rel="noreferrer nofollow"
        target="_blank"
      >
        <button>
          <p>
            Join the community on Instagram
          </p>
        </button>
      </a>
    </div>
  </div>
  <div
    className="landing-about"
  >
    <div
      className="about-content"
    >
      <p>
        Catcalls of Amsterdam is founded in December 2017, after Ambrien found the account Catcalls of New York on Instagram. After talking with Sophie Sandberg, its founder, she decided to join the movement and start chalking in Amsterdam.
      </p>
      <p>
        In the meantime our team grew and is organised by four people: Ambrien, Margriet, Paula and Zessiah. Together with 150 other initiatives worldwide, we 'chalk back'. Our goal is to provide a place to share experiences on street harassment and use people's stories to create awereness around catcalling and denormalize it.
      </p>
      <p>
        With this application outside of Instagram, we show our catcalling database on the map and make it easier for any victim to report their experience. Check it out and contact us if you have any questions.
      </p>
    </div>
    <div
      className="about-title"
    >
      About Catcalls of Amsterdam.
    </div>
  </div>
  <div
    className="footer"
  >
    <div
      className="footer-content"
    >
      <div
        className="footer-div"
      >
        <div
          className="footer-title"
        >
          ABOUT THIS MAP APPLICATION
        </div>
        <div>
          This is a project for Catcalls of Amsterdam by Lisanne Kraal. Inspired by the young activists, she contacted the organization to help them mapping their enormous database in an interactive way. This is a first version of a project under construction. Any feedback or want to contribute? Contact us.
        </div>
      </div>
      <div
        className="footer-div"
      >
        <div
          className="footer-title"
        >
          CONTACT
        </div>
        <div>
          Catcalls of Amsterdam
        </div>
        <div>
          <a
            href="https://www.instagram.com/catcallsofams/"
            rel="noreferrer nofollow"
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            <i
              className="fab fa-instagram"
            />
             PM on Instagram
          </a>
        </div>
        <div>
          <a
            href="mailto:catcallsofams@gmail.com"
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
          >
            <i
              className="far fa-envelope"
            />
             Send an email
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
